DemoApp/DemoApp/Views/Components/DatePickerView/DatePickerDemoView.swift

```diff
@@ -0,0 +1,58 @@
+import SwiftUI
+import SATSCore
+
+struct DatePickerDemoView: View {
+    @State var selectedDate: Date = Date()
+    @State var showFullCalendar: Bool = false
+
+    var body: some View {
+        DatePickerView(
+            selectedDate: $selectedDate,
+            showFullCalendar: $showFullCalendar,
+            viewData: generateViewData()
```

Comment:
maybe in this case is better to call this (at least from the invocation side) something else than view data.
what is this intended to be? available days?

The `generateViewData` method I assume it's something many users of this component would need, so I imagine there is a "common" implementation that can be exposed as a public static function

Reply:
Changed names, but will leave the "common" implementation for when it's needed. As of now we're getting all this data from the back-end, both for PT and GX.